<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestion</class>
 <widget class="QDialog" name="gestion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1031</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>991</width>
     <height>591</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>gestion client</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>971</width>
       <height>541</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="widget">
      <attribute name="title">
       <string>Ajouter</string>
      </attribute>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>55</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_id">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>50</y>
         <width>871</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>55</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>nom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>55</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>pr√©nom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>55</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>adresse</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>250</y>
         <width>55</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>tel</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>290</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Date de naissance </string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_nom">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>100</y>
         <width>871</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_prenom">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>140</y>
         <width>871</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_adresse">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>200</y>
         <width>871</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_tel">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>250</y>
         <width>871</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_DN">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>290</y>
         <width>821</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>370</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>ajoute</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Modifier</string>
      </attribute>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>55</y>
         <width>721</width>
         <height>231</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/img/images.jpg);</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="widget">
      <attribute name="title">
       <string>supprimer</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Afficher</string>
      </attribute>
      <widget class="QTableView" name="tabclient">
       <property name="geometry">
        <rect>
         <x>-15</x>
         <y>0</y>
         <width>941</width>
         <height>531</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
